security/compliance_operator/compliance-operator-crd.adoc

```diff
@@ -0,0 +1,509 @@
+<2> Specify how often the scan should be run in cron format.
+<3> Specify a list of scan specifications to run in the cluster.
+<4> Indicates the progress of the scans.
+<5> Indicates the overall verdict of the suite.
+
+The suite in the background creates the `ComplianceScan` object based on the `scans` parameter.
+You can programmatically fetch the `ComplianceSuites` events. To get the events for the suite, run the following command:
+[source,terminal]
+----
+$ oc get events --field-selector involvedObject.kind=ComplianceSuite,involvedObject.name=<name of the suite>
+----
+
+[IMPORTANT]
+====
+You might induce errors when you manually define the `ComplianceSuite` as it contains the XCCDF attributes.
+====
+
+[id="advance-compliance-scan-object_{context}"]
+=== Advanced ComplianceScan Object
+The Compliance Operator includes options for advanced users for the purpose of debugging or integration with existing tooling. While it is recommended that not to create a `ComplianceScan` object directly, instead you can manage it by a `ComplianceSuite` object.
+
+.Example Advanced `ComplianceScan` object
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ComplianceScan
+metadata:
+  name: <name of the scan>
+spec:
+  scanType: Node <1>
+  profile: xccdf_org.ssgproject.content_profile_moderate <2>
+  content: ssg-ocp4-ds.xml
+  contentImage: quay.io/complianceascode/ocp4:latest <3>
+  rule: "xccdf_org.ssgproject.content_rule_no_netrc_files" <4>
+  nodeSelector: <5>
+    node-role.kubernetes.io/worker: ""
+status:
+  phase: DONE <6>
+  result: NON-COMPLIANT <7>
+----
+
+<1> Specify either `Node` or `Platform`. Node profiles scan the cluster nodes and platform profiles scan the Kubernetes platform.
+<2> Specify the XCCDF identifier of the profile that you want to run.
+<3> Specify the container image that encapsulates the profile files.
+<4> It is optional. Specify the scan to run a single rule. This rule has to be identified with the XCCDF ID, and has to belong to the specified profile.
++
+[NOTE]
+====
+If you skip the `rule` parameter, then scan runs for all the available rules of the specified profile.
+====
+<5> If you are on the {product-title} and wants to generate a remediation, then nodeSelector label has to match the `MachineConfigPool` label.
++
+[NOTE]
+====
+If you do not specify `nodeSelector` parameter or match the `MachineConfig` label, scan will still run, but it will not create remediation.
+====
+<6> Indicates the current phase of the scan.
+<7> Indicates the verdict of the scan.
+
+[IMPORTANT]
+====
+If you delete a `ComplianceSuite` object, then all the associated scans get deleted.
+====
+
+When the scan is completed, it generates the result as Custom Resources of the `ComplianceCheckResult` object. However, the raw results are available in ARF format. These results are stored in a Persistent Volume (PV), which has a Persistent Volume Claim (PVC) associated with the name of the scan.
```

Comment:
Change to "When the scan is complete, ..."

Reply:
Done.